README.md

```diff
@@ -45,23 +47,51 @@ To grant access:
 Clone the repo to your development machine. 
 
 The project has two relevant Nuget packages:
-1. Microsoft.Azure.Services.AppAuthentication - makes it easy to fetch access tokens for Service-to-Azure-Service authentication scenarios. 
-2. Microsoft.Azure.ServiceBus - contains methods for interacting with Service Bus. 
+1. Azure.Identity - makes it easy to fetch access tokens for Service-to-Azure-Service authentication scenarios.
+2. Azure.Messaging.ServiceBus - contains methods for interacting with Service Bus. 
+
+```C# Snippet:ServiceBusAuthConnString
+// Create a ServiceBusClient that will authenticate using MSI.
+string nameSpace = "<service_bus_namespace>";
+ServiceBusClient client = new ServiceBusClient(nameSpace, new DefaultAzureCredential());
+```
 
-The relevant code is in WebAppServiceBus/WebAppServiceBus/Controllers/HomeController.cs file. The **Send** method will create an instance of a Service Bus QueueClient, configured to use a ManagedServiceIdentityTokenProvider. This TokenProvider's implementation uses the AzureServiceTokenProvider found in the Microsoft.Azure.Services.AppAuthentication library. AzureServiceTokenProvider will follow a set number of different methods, depending on the environment, to get an access token. Please refer to the AppAuthentication library's documentation for [local development authentication](https://docs.microsoft.com/en-us/azure/key-vault/service-to-service-authentication#local-development-authentication) to see more detail.
+Message sending is performed using the `ServiceBusSender`. Receiving is performed using the `ServiceBusProcessor`.
 
 ```csharp    
-var tokenProvider = TokenProvider.CreateManagedServiceIdentityTokenProvider();
-QueueClient sendClient = new QueueClient($"sb://{Config.Namespace}.servicebus.windows.net/", Config.Queue, tokenProvider);
-await sendClient.SendAsync(new Message(Encoding.UTF8.GetBytes(messageInfo.MessageToSend)));
-await sendClient.CloseAsync();
+string nameSpace = Config.Namespace;
```

Comment:
I think this code can be removed. We point out where the code is in the statement above so there isn't much need to repeat the code snippets in markdown here.